src/main/java/com/slimebot/commands/BugCommand.java

```diff
@@ -2,69 +2,213 @@
-			event.getMessage().delete().queue();
-			event.reply("Der Bug wurde erfolgreich geschlossen!").setEphemeral(true).queue();
-		}));
-	}
+public class BugCommand extends ListenerAdapter {
+
+    private static final Logger logger = LoggerFactory.getLogger(BugCommand.class);
+
+    private final Map<String, LocalDateTime> lastBugReport = new HashMap<>();
+
+    private static YamlFile getGitHubConfig(String guildId) {
+        YamlFile config = Config.getConfig(guildId, "githubConfig");
+        if (!config.exists()) {
+            createGitHubConfig(config);
+        }
+        try {
+            config.load();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        return config;
+    }
+
+    private static void createGitHubConfig(YamlFile config) {
+        try {
+            config.createNewFile();
+            config.set("enabled", false);
+            config.setComment("enabled", "Enable or disable the GitHub issue creation");
+            config.set("personalAccessToken", "");
+            config.set("repoName", "SlimeCloud/java-SlimeBot");
+            config.setComment("repoName", "The name of the repository where the issues should be created. Format: User/Repo");
+            config.set("user", "SlimeCloud");
+            config.setComment("user", "The user or organization that should be used to create the issues");
+            config.set("issueTemplate", """
+                    **Describe the bug**
+                    %description%
+                                        
+                         
+                    **To Reproduce**
+                    %steps%
+                                        
+                    					
+                    **Expected behavior**
+                    %expected%
+                                        
+                         
+                    **Solution idea**
+                    %solution%
+                                        
+                         
+                    **ToDos**
+                                        
+                         
+                    **Screenshots**
+                                        
+                         
+                    **Additional context**
+                    """);
+            config.setComment("issueTemplate", """
+                    %title% = Title of the issue
+                    %description% = Description of the issue
+                    %steps% = Steps to reproduce the issue
+                    %expected% = Expected behavior
+                    %solution% = Solution idea
+                    """);
+            config.save();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @ApplicationCommandMethod()
+    public void performCommand(SlashCommandInteractionEvent event) {
+        if (!event.getFullCommandName().equals("bug")) return;
```

Comment:
You don't have to manually check for that, DiscordUtils does that for you

Reply:
Done